.github/workflows/deploy.py

```diff
@@ -0,0 +1,150 @@
+import os
+
+import click
+import docker
+import sys
+import subprocess
+import requests
+import json
+from urllib.parse import urlparse
+
+try:
+    import click
+except ImportError:
+    print("Please install click library: pip install click==8.0.3")
+    sys.exit(1)
+
+
+ERR_MSG_TPL = {
+    "blocks": [
+        {
+            "type": "section",
+            "text": {"type": "mrkdwn", "text": ""},
+        },
+        {"type": "divider"},
+    ]
+}
+
+DOCKER_USER = os.environ.get("DHUBU")
```

Comment:
I think we can rename it to humanize view, DOCKER_USER, DOCKER_PASSWORD, like in other GHA repos

Reply:
@vodolaz can you rename please?